src/datumaro/plugins/data_formats/icdar/base.py

```diff
@@ -248,18 +248,19 @@ def _load_segmentation_items(self):
             gt_path = osp.join(self._path, item_id + "_GT" + IcdarPath.GT_EXT)
             if osp.isfile(gt_path):
                 # load mask through cache
-                mask = lazy_mask(gt_path, inverse_cls_colormap)
-                mask = mask()
+                index_mask = lazy_mask(gt_path, inverse_cls_colormap)
+                np_index_mask = index_mask()
 
-                classes = np.unique(mask)
+                classes = np.unique(np_index_mask)
                 for label_id in classes:
                     if label_id == 0:
                         continue
                     i = int(label_id)
                     annotations.append(
-                        Mask(
+                        ExtractedMask(
+                            index_mask=index_mask,
+                            index=label_id,
```

Comment:
We may miss the `label=label_id` here.

Reply:
It was intended. There was no `label` assignment previously. You can see L259, `i = int(label_id)` and L263 `group=groups[i]`. The original implementation just left a `group` assignment here.